### DISCUSSION TITLE
Finding object given a property

### DISCUSSION THREAD

**Discussion by fernfrost** [eemeli/yaml#290] [Q&A] [answered]
Hi

I have this object:
```
- subject: Test
  name: testSubject
  description: Some description
```
I do not know beforehand where or at what level in the tree it is located.
Any way of finding it based on the property "name"?

Read through the documentation but have not been able to figure it out.

Any help appreciated

**Comment by eemeli** [comment, marked as answer]
Try using a visitor: https://eemeli.org/yaml/#modifying-nodes
